src/Cache/Cache.php

```diff
@@ -191,17 +191,12 @@ protected static function _buildEngine(string $name): void
     }
 
     /**
-     * Fetch the engine attached to a specific configuration name.
-     *
-     * If the cache engine & configuration are missing an error will be
-     * triggered.
+     * Get a SimpleCacheEngine object for the named cache pool.
      *
-     * @param string $config The configuration name you want an engine for.
-     * @return \Cake\Cache\CacheEngine When caching is disabled a null engine will be returned.
-     * @deprecated 3.7.0 Use Cache::pool() instead. In 4.0 all cache engines will implement the
-     *   PSR16 interface and this method does not return objects implementing that interface.
+     * @param string $config The name of the configured cache backend.
+     * @return \Cake\Cache\CacheEngine
      */
-    public static function engine(string $config): CacheEngine
+    public static function pool(string $config)
```

Comment:
It might be nice to keep `engine()` as a deprecated alias for `pool` just in case folks are using it in their applications. It doesn't cost us anything to keep around a while longer.

Reply:
Done